src/FEValues/interface_values.jl

```diff
@@ -0,0 +1,265 @@
+# Defines InterfaceValues and common methods
+"""
+    InterfaceValues(quad_rule::FaceQuadratureRule, func_interpol::Interpolation, [geom_interpol::Interpolation])
+
+An `InterfaceValues` object facilitates the process of evaluating values, averages, jumps and gradients of shape functions
+and nodal functions on the interfaces of finite elements.
+
+**Arguments:**
+
+* `quad_rule`: an instance of a [`FaceQuadratureRule`](@ref)
+* `func_interpol`: an instance of an [`Interpolation`](@ref) used to interpolate the approximated function
+* `geom_interpol`: an optional instance of an [`Interpolation`](@ref) which is used to interpolate the geometry.
+  By default linear Lagrange interpolation is used.
```

Comment:
Is this really true? It seems like the default is the default interpolation of the respective geometry.